mautic/mautic
# Fixed Point actions applied on campaign

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #1518
| BC breaks? | 
| Deprecations? | 

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 

### Required
#### Description:
This PR checks all the configuration on a point action and applies it so that points are added correctly, when a point action is configured for visits a specific URL
#### Steps to reproduce the bug:
1. configure a point action that visits a URL
2. fill the criteria and visit a third party URL check points added
#### Steps to test this PR:
1. Apply this PR
2. try to reproduce the bug, point should be added correctly.
